packages/cli/src/handlers/createProject.ts

```diff
@@ -72,6 +75,7 @@ type CreateProjectHandlerOptions = {
 };
 export const createProject = async (
     options: CreateProjectHandlerOptions,
+    spinner?: ora.Ora,
```

Comment:
We can now take advantage of global state instead of passing the spinner around.

```const spinner = GlobalState.getActiveSpinner();```

Note that in deploy.ts we set the active spinner 
```GlobalState.setActiveSpinner(spinner);```